### PR TITLE
Image capture

### PR DESCRIPTION
**Pull request by littlestripes**
Adds image capture functionality. Firebase integration WIP

### CODE REVIEW

File: app/src/main/java/com/example/beetrackerrough/BeeSightingUserInput.kt [hunk 1]
```diff
@@ -1,13 +1,35 @@
 package com.example.beetrackerrough
 
+import android.annotation.SuppressLint
+import android.app.Activity.RESULT_OK
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
+import android.os.Environment
+import android.provider.MediaStore
+import android.telephony.MbmsDownloadSession.RESULT_CANCELLED
```
Comment (littlestripes) [inline app/src/main/java/com/example/beetrackerrough/BeeSightingUserInput.kt]: For some reason `android.app.Activity,RESULT_CANCELLED` doesn't exist, or according to Android Studio at least.

Comment (Jdono18) [reply app/src/main/java/com/example/beetrackerrough/BeeSightingUserInput.kt]: I think the request code is RESULT_CANCELED (with 1 L).  I updated it and will try running it.

**Comment by Jdono18** [comment]
I added an upload image to firebase feature.  Check it out and if it works for you then I think we can merge this into master.

**Review by Jdono18 — APPROVED** [review]
Code looks good to me.  Only problem I had was getting it running on API35.  I had to step down to API 34.

**Pull request merged**